environment.yaml

```diff
@@ -25,16 +26,18 @@ dependencies:
 - pytest=6.2.2
 - python=3.9.2
 - pyvirtualdisplay=1.3.2
+- recommonmark=0.7.1
 - redis-py=3.5.3
 - s3fs=0.5.2
 - selenium=3.141.0
 - sentry-sdk=0.20.3
+- sphinx-markdown-tables=0.0.15
+- sphinx=3.5.1
```

Comment:
We could run `repin.sh` again as `sphinx` and `nbsphinx` have new versions.

Reply:
Done